.prettierignore

```diff
@@ -1 +1,2 @@
 node_modules/
+doc/api
```

Comment:
This is not related to a performance improvement. Please revert this and open a separate PR/discussion.

Reply:
Doc files are ignored by git, but not by prettier, and prettier breaks on quite a few of them if you ran `npm run api-docs`. Why does this need discussion and a separate pr?

Reply:
It's part of our policy. Unrelated changes need to be in their own commits so our technical writers/product owners are able to create proper release notes. Because the majority of commits associated with this PR do not follow "conventional commits" guidelines, we'll be squashing this PR and therefore this change will land unnoticed.

As this change has nothing to do with performance, it belongs in a separate commit. I know this isn't something that will end up in release notes (as it's not consumer-facing), but if we let this slide, it's very likely we'll get in the habit of letting more "impactful" things slide too.

Sorry to be so strict here, but these are the rules set by the powers that be.

Reply:
FWIW I've had numerous discussions with several people about this and we may be implementing some process changes soon. 🤷‍♀